### DISCUSSION TITLE
How to change the order of the CV page?

### DISCUSSION THREAD

**Discussion by alexjameswilliams** [alshedivat/al-folio#2697] [Q&A] [answered]
The default ordering seems to be [Basics]
, [Work]
, [Volunteer]
, [Education]
 etc.
How would I change it to e.g.  [Basics], [Education], [Work], [Volunteer] ...

I can see how to remove sections under jsonresume in _confing.yml but this seems to have no bearing on the ordering.

**Comment by george-gca** [comment, marked as answer]
The ordering is the same as it appears on [assets/json/resume.json](https://github.com/alshedivat/al-folio/blob/master/assets/json/resume.json). Change the order there and it will change in the page.
